Java/SwapNumbersWithoutThirdVariable.java

```diff
@@ -0,0 +1,13 @@
+public class SwapNumbersWithoutThirdVariable {
+    public static void main(String[] args) {
+        int x = 9;
+        int y = 4;
+
+        x = x + y;
+        y = x - y;
+        x = x - y;
+
+        System.out.println(x);
+        System.out.println(y);
+    }
+}
```

Comment:
Please add sample I/O of your program as well as Space & Time complexities.

Reply:
@lostgirljourney Added both space & time complexities and sample I/o